dist/README.md

```diff
@@ -0,0 +1,20 @@
+# Releasing rules_proto
+
```

Comment:
I'd mention first that the release manager should check that the version number in `//proto:defs.bzl` is correct (since the previous one might have forgotten to update it).